inlong-manager/manager-client-tools/src/main/java/org/apache/inlong/manager/client/cli/util/ClientUtils.java

```diff
@@ -84,4 +85,17 @@ public static String readFile(File file) {
         return null;
     }
 
+    private static void initClientConfiguration() throws IOException {
+        Properties properties = new Properties();
+        String path = Thread.currentThread().getContextClassLoader().getResource("").getPath() + CONFIG_FILE;
+        InputStream inputStream = new BufferedInputStream(Files.newInputStream(Paths.get(path)));
```

Comment:
please use try-with-resource to close the stream

Reply:
done